src/components/sbb-menu/sbb-menu.tsx

```diff
@@ -191,6 +205,13 @@ export class SbbMenu implements ComponentInterface {
       return;
     }
 
+    this._triggerElement.setAttribute('aria-haspopup', 'menu');
+    this._triggerElement.setAttribute('aria-controls', this._element.id || this._menuId);
```

Comment:
True, but I need this logic here because in the `connectedCallback` I still don't have access to the potential auto-generated id assigned in the ref function.

Reply:
I see I see ;)